tests/integration/einstein/test_operations.py

```diff
@@ -13,6 +13,8 @@
 from ims.database.database import Database
 from ims.einstein.operations import BMI
 
+IMG2 = 'img2'
+
```

Comment:
Could you please move this to the chunk where EXIST_IMG_NAME, etc constants are defined ?

Reply:
You didnt address this